vuepress/docs/next/docs/reference/databases.md

```diff
@@ -60,29 +60,34 @@ the same namespace as the EntandoApp and EntandoPlugin that use them. It is usua
 
 `EntandoDatabaseService` custom resource example:
 
-     EntandoDatabaseService
-    metadata:
-      name: string, any K8s compliant name
-      namespace: string, the namespace this DB is created in
-    spec:
-      dbms: string, one of Oracle, PostgreSQL or MySQL
-      host: string, either an IP address or hostname where the database service is hosted
-      port: integer, the port on which the database service is hosted
-      databaseName: string, the name of the database, only required for PostgreSQL and Oracle
-      secretName: the name of the Secret in the same namespace carrying admin credentials to the database service
-      tablespace: (Oracle only)  the tablespace to use for required schemas 
-      jdbcParameters: a map containing name-value pairs for any additional parameters required for the JDBC driver to connect to the database
-
-An example of the Secret that provides the admin credentials, identified as
-`secretName` above:
-
-    Secret:
-      name: string, any K8s compliant name
-      namespace: string, the namespace the DB is created in
-      stringData:
-        username: string, name of an admin user that can create schemas and other users
-        password: string, password of the above user
+``` yaml
+kind: "EntandoDatabaseService"
+apiVersion: "entando.org/v1alpha1"
```

Comment:
Change to apiVersion: entando.org/v1. I checked in PS Dev and that's the current apiVersion for the EntandoDatabaseService.